release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3110,3 +3110,28 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.12.0 --pullspecs
 ----
 //replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
+[id="ocp-4-12-1"]
+=== RHSA-2023:0449 - {product-title} 4.12.1 bug fix and security update
+
+Issued: 2022-08-23
+
+{product-title} release 4.12.1, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0449[RHSA-2023:0449] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:0448[RHBA-2023:0448] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.1 --pullspecs
+----
+
+[id="ocp-4-12-1-bug-fixes"]
+==== Bug fixes
+
+* Previously, due to a wrong check in `cloud-provider-openstack`, the load balancers was popullating with External IP addresses once all of the Octavia load balancers get created. This caused an increase time for the load balancers to be handled. With this update, load balancers are still created sequentially and External IP addresses are populated one-by-one. (link:https://issues.redhat.com/browse/OCPBUGS-5403[*OCPBUGS-5403*])
+
+* Previously, `cluster-image-registry-operator` would default to using persistent volume claim (PVC) when it failed to reach Swift. With this update, failure to connect to {rh-openstack-first} API or other incidental failures cause `cluster-image-registry-operator` to retry the probe. During the retry, the default to PVC only occurs if the {rh-openstack} catalog is correctly found, and it does not contain object storage; or alternatively, if {rh-openstack} catalog is there and the current user does not have permission to list containers. (link:https://issues.redhat.com/browse/OCPBUGS-5154[*OCPBUGS-5154*])
```

Comment:
Slight revision: 

* Previously, the`cluster-image-registry-operator` would default to using persistent volume claims (PVC) when it failed to reach Swift. With this update, failure to connect to {rh-openstack-first} API or other incidental failures cause the `cluster-image-registry-operator` to retry the probe. During the retry, the default to PVC only occurs if the {rh-openstack} catalog is correctly found, and it does not contain object storage; or alternatively, if {rh-openstack} catalog is there and the current user does not have permission to list containers. (link:https://issues.redhat.com/browse/OCPBUGS-5154[*OCPBUGS-5154*])

Reply:
I only added an article here. The only other instance of this term in the docs uses "the"